LDAP-Auth/LDAPAuthenticationProviderPlugin.cs

```diff
@@ -106,9 +106,10 @@ public async Task<ProviderAuthenticationResult> Authenticate(string username, st
                         false);
 
                     // If we got non-zero, then the filter matched and the user is an admin
-                    if (ldapUsers.Count != 0)
+                    while (ldapUsers.HasMore())
                     {
                         ldapIsAdmin = true;
+                        break;
                     }
```

Comment:
```suggestion
                    if (ldapUsers.HasMore())
                    {
                        ldapIsAdmin = true;
                    }
```

Will have the same effect since we're not using the results from `ldapUsers`